pilot/PlatformProtocol.swift

```diff
@@ -11,8 +11,8 @@ import Foundation
 import HTTPStatusCodes
 
 enum PlatformType: String {
-  case twitter = "Twitter"
-  case facebook = "Facebook"
+  case twitter = "twitter"
+  case facebook = "facebook"
```

Comment:
You can just remove these strings since you're going back to lowercase. It will automatically use the case name as the string value.